### ISSUE TITLE
make clere that answeres lik " sorry i cant do that ..." are bad for the dataset.

### ISSUE THREAD

**Issue by Manni1000** [LAION-AI/Open-Assistant#1312]
make clear that answers like " sorry i cant do that ..." are bad for the datasets.
many don't know this.
you should just not answer them and mark the questions as "bad"

**Comment by blankster** [comment]
I think that is crucial. At the beginning, I was not aware that this explicitly not desired. I have just imitated what I just knew from ChatGPT and assumed that it's wanted to write texts like this.

The topic is certainly also generally connected with the general topic of guidelines. See also #1259

**Comment by Manni1000** [comment]
yes but violence or other stuff is only to be supost to be marked with the matching option. and not with a answer like this. but i also though like you at first
